### PR TITLE
add option to publish specific stations to a certain topic (#720)

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

**Comment by tomkralidis** [comment]
- remove hardcoded STATION path (overall)

**Comment by tomkralidis** [comment]
Update CI to run topic hierarchy addition in lieu of running station publication twice.

File: docs/source/user/setup.rst [hunk 1]
```diff
@@ -268,7 +268,7 @@ You can also bulk insert a set of stations from a CSV file, by defining the stat
 .. code-block:: bash
 
    python3 wis2box-ctl.py login
-   wis2box metadata station publish-collection
+   wis2box metadata station publish-collection /data/wis2box/metadata/stations/station_list.csv
 
 After doing a bulk insert please review the stations in wis2box-webapp and associate each station to the correct topics.
```
Comment (maaikelimper) [inline docs/source/user/setup.rst:273]: Can you please update the documentation to include the topic-association during the bulk-insert ? I am not sure what is the use-case for having station metadata in wis2box for stations that are not associated to WIS2-topic to be published, so adding a topic during the bulk-insert should be the baseline.

File: .github/workflows/tests-docker.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ jobs:
         curl -X POST http://localhost/wis2downloader/subscriptions -H "Content-Type: application/json" -H "Authorization: Bearer github123" -d @test.json
     - name: populate stations from CSV 📡
       run: |
-        python3 wis2box-ctl.py execute wis2box metadata station publish-collection
+        python3 wis2box-ctl.py execute wis2box metadata station publish-collection --path /data/wis2box/metadata/station/station_list.csv
```
Comment (maaikelimper) [inline .github/workflows/tests-docker.yml:58]: can we split the station-list into separate files per test dataset, remove the step "populate stations from CSV" and instead add one bulk-insert command for each dataset being tested ?

**Review by maaikelimper — CHANGES_REQUESTED** [review]

**Review by maaikelimper — APPROVED** [review]

**Pull request merged**